recipes-bsp/u-boot/u-boot-imx_2022.04.bb

```diff
@@ -16,7 +16,7 @@ UUU_BOOTLOADER_TAGGED     = ""
 UUU_BOOTLOADER_TAGGED:mx6-nxp-bsp = "u-boot-tagged.${UBOOT_SUFFIX}"
 UUU_BOOTLOADER_TAGGED:mx7-nxp-bsp = "u-boot-tagged.${UBOOT_SUFFIX}"
 
-do_deploy:append:mx8m-nxp-bsp() {
+do_deploy:append() {
```

Comment:
The right override to use is `mx8m-generic-bsp`.  All those need to be `generic-bsp` as the bootloader can also be used with mainline BSP.

Reply:
FIxed

Reply:
Please change all.

Reply:
Changed at all